internal/security/authz/authz.go

```diff
@@ -233,6 +378,31 @@ func Verify(t testing.TB, dut *ondatra.DUTDevice, spiffe *Spiffe, rpc *gnxi.RPC,
 			t.Errorf("Invalid option is passed to Verify function: %T", opt)
 		}
 	}
+	counterBefore := GrpcCounter{}
+	_, tempPolicy := Get(t, dut)
+	if !strings.Contains(spiffe.ID, "deny-all") {
+		if len(tempPolicy.DenyRules) > 0 && tempPolicy.DenyRules[0].Name != "no-one-can-gnmi" {
+			if rpc == gnxi.RPCs.GnsiAuthzProbe {
+				t.Logf("Probe")
+			}
+			data := gnmi.LookupAll(t, dut, gnmi.OC().System().GrpcServerAny().State())
+			sysGrpcServerName, pres := data[0].Val()
```

Comment:
is 'pres' here 'ok' ?

I suspect if you reach into a 'is not there' slice element you're going to have a 'panic' bad day.
Would it make sense to FIRST check:
  "if len(data) != 1 ....."
  
  regardless of the discussion (which is 100% valid) about the name/why '0' may work here, reaching into a thing that maybe doesn't exist is always a bad plan.